src/ftxui/component/screen_interactive.cpp

```diff
@@ -147,8 +147,12 @@ void ScreenInteractive::Loop(Component* component) {
 
   // Handle resize.
   on_resize = [&] { PostEvent(Event::Special({0})); };
+#ifdef WIN32
+  // TODO: port
+#else
   auto old_sigwinch_handler = std::signal(SIGWINCH, OnResize);
```

Comment:
I am unsure how this is handled on Windows but I think just handling the `WINDOW_BUFFER_SIZE_EVENT` is the way to implement this.

Reply:
this is now fixed